src/utils/midi.ts

```diff
@@ -41,6 +44,8 @@ export async function playNote(note: MidiNote) {
     if (!(await loadInstrument(note.name))) {
```

Comment:
This return value probably shouldn't be checked? Is it awaiting `undefined` now or am I missing something?